apisix/utils/log-util.lua

```diff
@@ -52,6 +52,10 @@ local function get_full_log(ngx, conf)
             headers = ngx.resp.get_headers(),
             size = var.bytes_sent
         },
+        server = {
+            hostname = core.utils.gethostname(),
+            version = core.version.VERSION
```

Comment:
I think it would be better to use `apisix_version` so that it is easier to understand its meaning.

Reply:
IMHO, since this log comes from APISIX, we don't need a prefix to distinguish it.